changelog/12027.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+secrets/gcp: Document the ability to use existing service accounts for generation of service account keys and access tokens.
```

Comment:
We don't usually write CL entries for documentation, so we can remove this file. I think the entry in 12023.txt is good enough.

Reply:
OK. But if I remove this file one of the tests will fail. That won't be an issue right?

Reply:
I've added a `pr/no-changelog` label so that the test will pass 👍

Reply:
Got it. Removed the file.